english/malaria/malaria.md

```diff
@@ -1,21 +1,23 @@
 The hand of marshmallow is speeding towards you and growing as it approaches
 . You start running away from it, but it seems that the walls are closing
  in on you. Suddenly everthing stops and dead silence ensues. You turn
-around to take a look at the marshmallow hand and in a flash you are 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.